Gemfile

```diff
@@ -10,6 +10,5 @@ gemspec
 # Git. Remember to move these dependencies to your gemspec before releasing
 # your gem to rubygems.org.
 
-ruby '~> 2.6.2'
-
+gem "rails", "~> 6.0"
```

Comment:
This is the default version of rails that will be run if you don't specify anything different when running the tests locally.